launcher/settingsView/csettingsview_moc.cpp

```diff
@@ -228,13 +246,36 @@ void CSettingsView::fillValidResolutionsForScreen(int screenIndex)
 
 	ui->comboBoxResolution->blockSignals(false);
 }
+
+void CSettingsView::fillValidRenderers()
+{
+	ui->comboBoxRendererType->blockSignals(true); // avoid saving wrong resolution after adding first item from the list
```

Comment:
use QSignalBlocker

Reply:
huh, wasn't aware of it. Replaced where applicable.